### DISCUSSION TITLE
essay/2021/09/14/%EC%A0%9C%EA%B0%80-%EC%84%9C%EB%B2%84%EB%9E%91-%ED%81%B4%EB%9D%BC%EC%9D%B4%EC%96%B8%ED%8A%B8%EB%9E%91-%EB%8B%A4-%ED%95%A0%EA%B2%8C%EC%9A%94

### DISCUSSION THREAD

**Discussion by giscus** [BenjaminKim/benjaminkim.github.io#199] [Comments]
# essay/2021/09/14/%EC%A0%9C%EA%B0%80-%EC%84%9C%EB%B2%84%EB%9E%91-%ED%81%B4%EB%9D%BC%EC%9D%B4%EC%96%B8%ED%8A%B8%EB%9E%91-%EB%8B%A4-%ED%95%A0%EA%B2%8C%EC%9A%94

저는 윈도우 클라이언트 프로그래머로 첫 경력을 시작했습니다. 회사에 들어왔을 때 처음 사용했던 툴은 비주얼 스튜디오 2005였습니다.

https://jeho.page/essay/2021/09/14/%EC%A0%9C%EA%B0%80-%EC%84%9C%EB%B2%84%EB%9E%91-%ED%81%B4%EB%9D%BC%EC%9D%B4%EC%96%B8%ED%8A%B8%EB%9E%91-%EB%8B%A4-%ED%95%A0%EA%B2%8C%EC%9A%94.html

**Comment by pabami** [comment]
멋집니다!
